app/routes/app/AppRoute.js

```diff
@@ -201,32 +201,27 @@ const mapDispatchToProps = {
 };
 
 export default compose(
-  prepare((_, dispatch) => dispatch(loginAutomaticallyIfPossible()), [], {
-    componentDidMount: false,
-    componentWillReceiveProps: false,
-  }),
-  prepare((_, dispatch) => dispatch(fetchMeta()), [], {
-    componentDidMount: false,
-    componentWillReceiveProps: false,
-    awaitOnSsr: false,
-  }),
```

Comment:
How has this changed now? if at all?
I would assume that this never refetches? Even on re-mount, but it might be hard to test considering the AppRoute never unmounts?

Reply:
Hmm, ok this one is interesting. I previously assumed it would not change as it was never run on the client before, and shouldn't now, but I didn't think of the case where we do not server-side render.

These prepare-functions previously never ran on client at all. However the same actions are dispatched when initializing the store on the client when _not_ server-side rendered. 

Now it will also dispatch them from the `withPreparedEffect` whenever we do not server-side render, which might not be great. I believe the behaiviour we want is basically the default for the new hooks, only run on client if not ran on the server. I will look into this further.

Reply:
It will also re-run on re-mount, but App should never unmount so that shoudn't matter.

Reply:
It should now work exactly as it did before:)